ksqldb-rest-app/pom.xml

```diff
@@ -155,7 +155,7 @@
 
         <dependency>
             <groupId>com.kjetland</groupId>
-            <artifactId>mbknor-jackson-jsonschema_2.12</artifactId>
```

Comment:
We should use the scala version variable here. @purplefox your PR doesn't seem to fix this, so why did we close this PR?